test/e2e/daemon_set.go

```diff
@@ -445,11 +509,19 @@ func getDaemonSetImagePatch(containerName, containerImage string) string {
 	return fmt.Sprintf(`{"spec":{"template":{"spec":{"containers":[{"name":"%s","image":"%s"}]}}}}`, containerName, containerImage)
 }
 
-func orphanDaemonSetPods(c clientset.Interface, ds *extensions.DaemonSet) error {
+func orphanDaemonSetPods(c clientset.Interface, ds *extensions.DaemonSet) {
```

Comment:
This is not orphaning pods only anymore. `deleteDaemonSet` or `cascadeDeleteDaemonSet` or something with more context.